catalog/transformer/catalog-transformer-pdf/src/test/groovy/ddf/catalog/transformer/input/pdf/PdfInputTransformerSpecTest.groovy

```diff
@@ -55,16 +74,9 @@ class PdfInputTransformerSpecTest extends Specification {
 
     void verifySamplePdfMetacard(Metacard metacard) {
         verifyThumbnailSize(metacard.thumbnail)
-        assert metacard.contentTypeName == 'pdf'
-        assert metacard.title == 'Microsoft Word - Document1'
```

Comment:
by default, the input transformer does not set the title. There are separate test method in PdfInputTransformerTest that tests the title functionality.